data/pg/inbox-v2/InboxConfiguration.json

```diff
@@ -276,6 +276,48 @@
         "defaultOrder": "DESC"
       },
       "sourceFilterPathList": ["Data.tenantId" ,"Data.id", "Data.billNumber", "Data.totalAmount",  "Data.additionalDetails", "Data.businessService","Data.currentProcessInstance", "Data.auditDetails","Data.referenceId"]
+    },
+    {
+      "module": "measurement-service",
+      "index": "measurement-service-inbox",
```

Comment:
update the index name
https://github.com/egovernments/configs/blob/e5e776e0f37bf9474d8ff780176e7f4eb16d65ee/works/egov-indexer/measurement-indexer.yml#L8